### PR TITLE
Add new a section for related lists to this awesome list #1338

### PR DESCRIPTION
**Pull request by PaulRosset**
Hello,

Following a discussion #1338, I've created a new section where we can store all the lists that are related to the awesome lists.

### CODE REVIEW

**Comment by inyutin** [comment]
Cool! I think we can merge this

**Comment by sindresorhus** [comment]
Sorry, I missed this PR. There's now a section you can add it to: https://github.com/sindresorhus/awesome#related

File: readme.md [hunk 1]
```diff
@@ -584,6 +585,9 @@
 - [Calculators](https://github.com/xxczaki/awesome-calculators#readme) - Calculators for every platform.
 - [Captcha](https://github.com/ZYSzys/awesome-captcha#readme) - A type of challenge–response test used in computing to determine whether or not the user is human.
 
+## Awesome related lists
+
+- [Awesome lists, search as a service](https://awesome.paulrossethings.com/) - Navigating between Awesome list has never been so easy!
```
Comment (sindresorhus) [inline readme.md]: The title looks to be `Awesome Pretty List`.

The description should describe what it does and not be a marketing tagline.

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Pull request closed without merging**